fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/boot/EmbeddedTomcatWithSSLConfiguration.java

```diff
@@ -55,7 +55,7 @@ protected Connector createSslConnector() {
             File keystore = getFile(getKeystore());
             connector.setScheme("https");
             connector.setSecure(true);
-            connector.setPort(getHTTPSPort());
+            connector.setPort(getHTTPSport());
```

Comment:
Would getHttpsPort not make more sense - now it looks like getHttpSport :-)

Reply:
fixed